### PR TITLE
Migration integration-service to konflux-ci org

### PR DESCRIPTION
**Pull request by MartinBasti**

### CODE REVIEW

**Review by dirgim — APPROVED** [review]
LGTM

File: examples/task.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ spec:
       name: NAMESPACE
       description: Namespace of the application under test
   steps:
-    - image: quay.io/redhat-appstudio/hacbs-test:latest
+    - image: quay.io/konflux-ci/konflux-test:latest
```
Comment (jsztuka) [inline examples/task.yaml]: Just a question, do we know when those repos in quay will be migrated also?

**Review by jsztuka — APPROVED** [review]
lgtm

File: examples/task.yaml (hunk 1 above)
Comment (MartinBasti) [reply examples/task.yaml]: uff this is wrong, this shouldn't be migrated

Comment (MartinBasti) [reply examples/task.yaml]: fixed to `redhat-appstudio/konflux-test` for images we are keeping old org, only image name changed

**Pull request merged**